recipes/libnvjpeg/meta.yaml

```diff
@@ -0,0 +1,147 @@
+{% set name = "libnvjpeg" %}
+{% set version = "12.0.0.28" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: ad3380591a4d5b97ab57d10e9bf05e23dee6429ff91ea939423a3b35ee0e35de  # [linux64]
+  sha256: f7136bb691d794c680d77376b306bedb80eeebf687bfa4dc74355fdae5adcebb  # [ppc64le]
+  sha256: 31fd6ee0dccc72f0d583e25b849543551eb4bc49d3285628bf3b17929e09bac1  # [aarch64]
+  sha256: b7cb56ab2c69e055a4b1ea41653df917153a568ab23fb0d8121e0739011989a8  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+test:
+  commands:
+    - test -L $PREFIX/lib/libnvjpeg.so.{{ version.split(".")[0] }}                            # [linux]
+    - test -L $PREFIX/lib/libnvjpeg.so.{{ version }}                                          # [linux]
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libnvjpeg.so.{{ version.split(".")[0] }}  # [linux]
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libnvjpeg.so.{{ version }}                # [linux]
+    - if not exist %LIBRARY_BIN%\nvjpeg64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+
+outputs:
+  - name: libnvjpeg
+    files:
+      - lib/libnvjpeg.so.*            # [linux]
+      - targets/{{ target_name }}/lib/libnvjpeg.so.*  # [linux]
+      - Library\bin\nvjpeg*.dll       # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    # Tests are defined at the top level, due to package/output name conflicts.
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: NVJPEG native runtime libraries
```

Comment:
For consistency with requests I saw on some of the other recipes, perhaps worth doing a find and replace `s/NVJPEG/nvJPEG/g`.